install/kubernetes/values.yaml

```diff
@@ -171,6 +171,11 @@ tetragonOperator:
     # -- Enables the PodInfo CRD and the controller that reconciles PodInfo
     # custom resources.
     enabled: false
+  # -- Enables the Tetragon Operator metrics.
+  metrics:
+    enabled: true
+    # -- The port at which to expose metrics.
+    port: 2113
```

Comment:
At the moment the [metrics port in the operator](https://github.com/cilium/tetragon/blob/3326a78fbb120a9f5946b64733f8786d08be4b0e/operator/cmd/serve/serve.go#L93) is by default set to 8080. The Helm chart default should be same IMO, but `2113` seems to me a better choice for the default, so maybe let's change it in the operator.

Reply:
thanks, let's go with `2113`